benchmarks/shelley3pools/run-Npools.sh

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bash
+
+. ./configuration/parameters
+NODECMD="../../bin/cardano-node"
+
+# the host address and interface the node listens:
+HOSTADDR=127.0.0.1
+
+# the nodes will listen on ports starting with:
+PORTBASE=3000
+
+GENESISDIR=configuration/genesis
+
+# redirect stderr if LiveView active
+REDIRSTDERR="2>/dev/null"
+REDIRSTDERR=
+
+TMUX_ENV_PASSTHROUGH=(
+         "export SCRIPTS_LIB_SH_MODE=${SCRIPTS_LIB_SH_MODE};"
+         "export __COMMON_SRCROOT=${__COMMON_SRCROOT};"
+         "export DEFAULT_DEBUG=${DEFAULT_DEBUG};"
+         "export DEFAULT_VERBOSE=${DEFAULT_VERBOSE};"
+         "export DEFAULT_TRACE=${DEFAULT_TRACE}"
+         "$(nix_cache_passthrough)"
+)
+tmux split-window -v
+tmux split-window -h
+tmux select-pane -t 0
+tmux split-window -h
```

Comment:
let's put at most four nodes in a tmux windows, then create another one for the next four nodes and so on.